project/depends.scala

```diff
@@ -0,0 +1,22 @@
+import sbt._
+
+object depends {
+
+  def scalatest(scalaVersion: String) =
+    if (scalaVersion.startsWith("2.12")) "org.scalatest" %% "scalatest" % "3.0.0" % Test
+    else "org.scalatest" %% "scalatest" % "2.2.6" % Test
+
+  def akkaActor(scalaVersion: String) =
+    if (scalaVersion.startsWith("2.12")) "com.typesafe.akka" %% "akka-actor" % "2.4.8"
+    else if (scalaVersion.startsWith("2.11")) "com.typesafe.akka" %% "akka-actor" % "2.4.8"
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Consider using case matching instead of else if blocks](https://www.codacy.com/app/pathikrit/better-files/file/3776472232/issues/source?bid=2860326&fileBranchId=3676352#l11)